BrickKit/app/src/main/java/com/wayfair/brickkitdemo/mainfragment/MainActivityFragment.kt

```diff
@@ -30,7 +30,7 @@ class MainActivityFragment : BrickFragment() {
     override fun onCreate(savedInstanceState: Bundle?) {
         super.onCreate(savedInstanceState)
 
-        val placeholderBrick = ActiveBrick(resources, R.string.empty) { changeFragment(SimpleBrickFragment()) }
+        val placeholderBrick = ActiveBrick(resources, R.string.empty) { changeFragment(SimpleBrickFragment.newInstance(100)) }
```

Comment:
why 100? should it be a constant?

Reply:
this was unnecessary so I removed it (but made the underlying value a constant)